Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java

```diff
@@ -115,6 +115,15 @@ private static String readFile(@NonNull final File file) throws IOException {
                         "&cSkipping PlotSquared.use_THIS.yml because it's empty");
                 }
 
+                try {
```

Comment:
The command description should be altered to mention this file as well, to not cause any further issues with people claiming we steal their data. Last time wasn't very fun.